docs/PartRendering.md

```diff
@@ -0,0 +1,52 @@
+# Rendering of parts
+
+## Server side and client side rendering
+
+Due to a bug in react4xp we had to render all parts on the client side in order for them to work. The bug has now been fixed, and we can properly set up serverside rendering for all parts.
```

Comment:
Suggestion: (...) we had to client side render all the parts, as they would behave unpredictably. Most if not all parts would come and go. That issue has been resolved so we can properly set up serverside rendering for all the parts now.